phoenix-core-client/src/main/java/org/apache/phoenix/util/MetaDataUtil.java

```diff
@@ -1182,4 +1184,24 @@ public static void deleteFromStatsTable(PhoenixConnection connection,
             connection.setAutoCommit(isAutoCommit);
         }
     }
+
```

Comment:
Maybe we can move this function to metadataclient class only since that is where everything is being used